cmd/scheduler/entrypoints/precheck.go

```diff
@@ -0,0 +1,83 @@
+package entrypoints
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/flyteorg/flytestdlib/logger"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/metadata"
+	"google.golang.org/grpc/status"
+
+	"github.com/avast/retry-go"
+	adminClient "github.com/flyteorg/flyteidl/clients/go/admin"
+	"github.com/pkg/errors"
+	healthpb "google.golang.org/grpc/health/grpc_health_v1"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	timeout = 30 * time.Second
+)
+
+var preCheckRunCmd = &cobra.Command{
+	Use:   "precheck",
+	Short: "This command will check pre requirement for scheduler",
+	RunE: func(cmd *cobra.Command, args []string) error {
+		opts := []grpc.DialOption{
+			grpc.WithUserAgent("grpc_health_probe"),
+			grpc.WithBlock(),
+			grpc.WithInsecure(),
+		}
+
+		err := retry.Do(
+			func() error {
+				ctx := context.Background()
+				config := adminClient.GetConfig(ctx)
```

Comment:
What happens when admin is configured with auth and ssl

Reply:
Currently auth is not tested and SSL will be handle by the ingress